cg/services/order_validation_service/workflows/rml/models/sample.py

```diff
@@ -0,0 +1,12 @@
+from cg.models.orders.sample_base import ControlEnum, PriorityEnum
+from cg.services.order_validation_service.models.sample import Sample
+
+
+class RmlSample(Sample):
```

Comment:
Minor thing - for RML the volume is a mandatory field. We should consider overriding the volume attribute set on the parent class to mark it as non-optional here